invenio/modules/workflows/models.py

```diff
@@ -574,7 +574,7 @@ def get_action(self):
     def get_action_message(self):
         """Retrieve the currently assigned widget, if any."""
         try:
-            return self.get_extra_data()["_message"]
+            return unicode(self.get_extra_data()["_message"])
```

Comment:
@jalavik please use unicodifier or other "safe" solution that can be easily migrated to PY2/3 support. Thanks